api/src/util/api_key_gen.py

```diff
@@ -0,0 +1,7 @@
+import secrets
+import string
+
+
+def generate_api_key_id(length: int = 25) -> str:
+    alphabet = string.ascii_letters + string.digits  # a-z, A-Z, 0-9
+    return "".join(secrets.choice(alphabet) for _ in range(length))
```

Comment:
However unlikely, I don't think this guarantees a key will be unique. We'll need to check that it doesn't already exist in the DB where we use this.

Actually, double-checking the user_api_key schema, we don't enforce that key_id should be unique + indexed, which we should be doing.